appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -126,37 +126,24 @@ private fun BoxScope.ComposeLastRenderingState(state: Rendering) {
             // apply backdrop traits
             ApplyBackgroundDecoratingTraits(backdropDecoratingTraits.value)
 
-            // TODO contentPadding + stepDecoratingPadding into one? to fix tooltip padding issue
             // create wrapper
-            contentWrappingTrait.WrapContent { hasFixedHeight, contentPadding ->
+            contentWrappingTrait.WrapContent { modifier, wrapperInsets ->
```

Comment:
nothing much changed here, basically simplified this part of the composition, and moved some code to ComposeStep, added the wrapperInsets  as the result of safe area that is available for the step content and decorators